source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/PeriodCreation.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@model CommonJobs.Mvc.UI.Areas.Evaluations.Models.PeriodCreation
+@{
+    ViewBag.Title = "Generate Evaluations";
+    Layout = "~/Areas/Evaluations/Views/Shared/_Layout.cshtml";
+    ScriptManager.RegisterCss(Url.Content("~/Content/twitter-bootstrap/less/bootstrap.less"), priority: 900);
+    ScriptManager.RegisterCss(Url.Content("~/Content/twitter-bootstrap/less/responsive.less"), priority: 900);
+    ScriptManager.RegisterCss(Url.Content("~/Content/Evaluations/evaluations.css"));
+    ScriptManager.RegisterGlobalJavascript("baseUrl", Url.Content("~"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/jquery-1.7.2.min.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/underscore.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Content/twitter-bootstrap/js/bootstrap-typeahead.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Content/twitter-bootstrap/js/bootstrap-modal.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/url-generator.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/knockout-2.2.0.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/evaluations-common.js"));
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/evaluations-admin.js"));
+}
+
+<div class="content-evaluation generate-evaluation">
+    <h2 class="title">Generar Evaluaciones</h2>
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Empleado</th>
+                <th>Posición</th>
+                <th>Seniority</th>
+                <th>Responsable</th>
+            </tr>
+        </thead>
+        <tbody data-bind="foreach: employees">
+            <tr data-bind="css: { 'evaluation-generated': period }">
+                <td data-bind="text: fullName"></td>
+                <td data-bind="text: currentPosition"></td>
+                <td data-bind="text: seniority"></td>
+                <td>
+                    <span data-bind="visible: period, text:responsible"></span>
+                    <input class="evaluation-owner-field" data-bind="visible: !period, value: responsible" autocomplete="off"/>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <button type="submit" id="generate-evaluation-button" class="button-evaluations generate-evaluation" data-bind="enable: generateButtonEnable">GENERAR EVALUACIONES</button>
```

Comment:
I would leave this in sentence case (`Generar Evaluaciones`) and show it in uppercase with CSS. Just a detail.